Project-Descriptions-and-Plans/LLVM-Toolchain/COREV_LLVM_Plan_Approved_2023.md

```diff
@@ -0,0 +1,173 @@
+# OpenHW Project: corev-llvm-project Plan Approved CV32E40P-V2
+
+| Gate                                 | Status                                                     |
+| ------------------------------------ | ---------------------------------------------------------- |
+| PC gate: Project Concept             | Approved on 2020-11-21 as Preliminary Project Launch (PPL) |
+| PL gate: Project Launch              | Approved on 2022-xx-x                                     |
+| PA gate: Plan Approved               | Presented on 2022-11-28 for **2023 workplan**, approved on 2023-xx-xx |
+| PA gate: Plan Approved               | 2024 and beyond workplan in future PA gate |
+
+Author: ChunyuLiao
+
+## Summary of project
+
+This document should be read in the context of the overall CORE-V LLVM project project launch proposal.  It does not duplicate information from that document.
+
+This plan is to deliver Clang/LLVM compiler support for all of the CV32E40P version 2 instruction set extensions.
+
+### Summary of Timeline
+
+CORE-V LLVM project is a long-run project. It is therefore difficult to plan the complete project.
+
+Therefore, in order to keep the project under control, the project plan is defined for 2023 activity. 
```

Comment:
However, as this is the PA for the CV32E40Pv2 can you provide an overview of the timeline for what LLVM features will be completed for that processor in 2023? I understand there may be ongoing maintenance etc. in future

Reply:
done